env/production/locals.tf

```diff
@@ -7,6 +7,8 @@ locals {
   # our needs.
   #   -trs, 20 May 2024
 
+  # These are pathogen repos that are using pathogen-repo-build. There are other
+  # pathogen repos such as seasonal-cov that are not included here.
```

Comment:
I assume `seasonal-cov` will be added soon-ish as part of https://github.com/nextstrain/seasonal-cov/issues/14. 

Instead of pointing to a specific example repo that might be outdated, maybe okay to be a bit vague here

```Suggestion
  # These are pathogen repos that are using pathogen-repo-build. There are other
  # pathogen repos that have not been automated yet that are not included here.
```

Reply:
Done in [force-push](https://github.com/nextstrain/infra/compare/d5a78a3db65737beb4acc228f6e387c0e97c666c..5eda37f4597f0beaae01763d6879bd0a9a1c0cdd)